certified-connectors/Redque/apiDefinition.swagger.json

```diff
@@ -1,4 +1,3551 @@
+        "CONTAINER_NOT_FOUND",
+        "CONTAINER_NOT_REMOVED",
+        "DOCUMENT_NOT_RETURNED",
+        "DOCUMENT_RETURNED_NOT_SAVED",
+        "DOCUMENT_NOT_READY",
+        "DOCUMENT_UNSUPPORTED_TYPE",
+        "DOCUMENT_EXTRACT_RESULT_NOT_FOUND",
+        "DOCUMENT_FILE_NOT_FOUND",
+        "DOCUMENT_FILE_UPLOAD_FAILED",
+        "DOCUMENT_FORBIDDEN_TYPE",
+        "ATTACHMENT_NOT_ATTACHED",
+        "PASSWORD_NOT_ACCEPTED",
+        "EMAIL_NOT_FOUND",
+        "NO_TEXT_DETECTED",
+        "ENTITY_DELETE_FAILED"
+      ],
+      "type": "string"
+    },
+    "SecureSocketOptions": {
+      "description": "Secure socket options.",
+      "enum": [
+        "None",
+        "Auto",
+        "SslOnConnect",
+        "StartTls",
+        "StartTlsWhenAvailable"
+      ],
+      "type": "string"
+    },
+    "StringApiFilterEqual": {
+      "description": "Equal filter implementation",
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "string"
+        }
+      }
+    },
+    "StringApiFilterMultipleChoice": {
+      "description": "Multiple choice filter implementation",
+      "type": "object",
+      "properties": {
+        "values": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "StringApiFilterNotEqual": {
+      "description": "Equal filter implementation",
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "string"
+        }
+      }
+    },
+    "StringApiListValueUpdate": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "string"
+        },
+        "operation": {
+          "$ref": "#/definitions/ApiListValueOperation"
+        }
+      }
+    },
+    "StringApiValue": {
+      "description": "Generic response value",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Value",
+          "type": "string"
+        }
+      }
+    },
+    "StringApiValueUpdate": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "string"
+        }
+      }
+    }
+  },
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://redque.cz/en/"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://redque.cz/policy.html"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "AI;Finance"
+    }
+  ],
+  "schemes": [
+    "https"
+  ],
+  "security": [
+    {
+      "oauth2_auth": []
+    }
+  ],
+  "securityDefinitions": {
+    "oauth2_auth": {
+      "authorizationUrl": "https://demoidentity.redque.com/connect/authorize",
+      "flow": "accessCode",
+      "scopes": {
+        "v1.public offline_access": "v1.public offline_access"
+      },
+      "tokenUrl": "https://demoidentity.redque.com/connect/token",
+      "type": "oauth2"
+    }
+  }
+=======
```

Comment:
@jirisodomka it looks like you have the swagger file duplicate here, can you please fix?